Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+pipeline {
+    agent any
+
+    tools {
+        maven 'apache-maven-latest'
+        jdk 'openjdk-jdk11-latest'
+    }
+
+    stages {
+        stage ('Build Checkstyle project') {
+            steps{
+               dir('codestyle/org.eclipse.emfcloud.checkstyle'){
+                    sh 'mvn clean verify ' 
+                }
+            }
+        }
+     
+        stage('Deploy') {
+            when { 
+                allOf {
+                    branch 'master';
+                    changeset  pattern: "codestyle/org.eclipse.emfcloud.checkstyle", comparator: "REGEXP";  
```

Comment:
Could you please add some comment here to explain what the goal of that line is. To me it is not completely obvious :-)